CHANGES.rst

```diff
@@ -12,6 +12,8 @@ Other Changes and Additions
 Bug Fixes
 ^^^^^^^^^
 
+- ``test_image_collection.py`` in the test suite no longer produces
+ permanent files on disk and cleans up after itself. [#736]
```

Comment:
Please also add the PR number here

Reply:
I pushed the changes, do I have to do anything to this PR?